play/benchmarker/public/javascripts/hello.js

```diff
@@ -0,0 +1,3 @@
+if (window.console) {
+  console.log("Welcome to your Play application's JavaScript!");
```

Comment:
Yeah. This doesn't actually get served anywhere, but it may affect compile time.